tests/app/template/test_rest.py

```diff
@@ -616,6 +616,37 @@ def test_update_set_process_type_on_template(client, sample_template):
     assert template.process_type == 'priority'
 
 
+def test_create_a_template_with_reply_to(client, sample_user):
+    service = create_service(service_permissions=['letter'])
+    letter_contact = create_letter_contact(service, "Edinburgh, ED1 1AA")
+    data = {
+        'name': 'my template',
+        'subject': 'subject',
+        'template_type': 'letter',
+        'content': 'template <b>content</b>',
+        'service': str(service.id),
+        'created_by': str(sample_user.id),
+        'reply_to': str(letter_contact.id),
+    }
+    data = json.dumps(data)
+    auth_header = create_authorization_header()
+
+    response = client.post(
+        '/service/{}/template'.format(service.id),
+        headers=[('Content-Type', 'application/json'), auth_header],
+        data=data
+    )
+    assert response.status_code == 201, response.get_data(as_text=True)
+
+    json_resp = json.loads(response.get_data(as_text=True))
```

Comment:
if you use the `admin_request` fixture, it does a lot of the gubbins around setting up a request - json handling, headers, URL creation, etc.

```python
json_resp = admin_request.post('template.template', service_id=service.id, _data=data, _expected_status=201)
```